protecode/scanning_util.py

```diff
@@ -290,15 +294,18 @@ def enumerate_reference_triages():
 
         if self._processing_mode is ProcessingMode.FORCE_UPLOAD:
             logger.info('force-upload - will re-upload all images')
-            images_to_upload += list(resource_group.resources())
+            for upload_element in upload_elements:
+                elements_to_upload.append(upload_element)
```

Comment:
this loop seems a bit inconvenient

Reply:
true - it does not improve readability compared to the previous version (might be slightly more efficient, though). also, since this does not change the semantics, I think this ought to be part of a refactoring-commit